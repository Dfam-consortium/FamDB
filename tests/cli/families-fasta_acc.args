families
--format
fasta_acc
--add-reverse-complement
-a
3
