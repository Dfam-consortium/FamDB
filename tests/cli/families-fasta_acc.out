>TEST0001.1 name=Test family TEST0001 @root
ACGT
>TEST0001.1 (anti) name=Test family TEST0001 @root
ACGT
>TEST0003.1 name=Test family TEST0003 @Third_Clade [S:30,40]
GGTC
>TEST0003.1 (anti) name=Test family TEST0003 @Third_Clade [S:30,40]
GACC
